scapy/supersocket.py

```diff
@@ -389,6 +390,10 @@ def close(self):
         SuperSocket.close(self)
         self.tcpdump_proc.kill()
 
+    @staticmethod
```

Comment:
Why that? This is going to have massive performance impacts. Or am I missing something?

Reply:
This should now only impact windows and darwin, on which the select call wouldn't work anyways